### PR TITLE
update copyright year

### PR DESCRIPTION
**Pull request by cstyan**
update year in copyright footer from 2016 -> 2017

### CODE REVIEW

File: layouts/container_footer.html [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 <footer>
   <p class="pull-left">
-    &copy; Prometheus Authors 2016
+    &copy; Prometheus Authors 2017
```
Comment (grobie) [inline layouts/container_footer.html]: Alright, let's solve this issue once and for all with this PR: `<%= Date.today.year %>`.

Comment (cstyan) [inline layouts/container_footer.html]: good idea! I can make that change

out of curiosity, what templating language are we using?

Comment (grobie) [inline layouts/container_footer.html]: We use https://nanoc.ws/, a ruby static site generator. All HTML files are run through an [ERB filter](https://github.com/prometheus/docs/blob/master/Rules#L43).

**Comment by cstyan** [comment]
@grobie updated

File: test.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+require 'date'
```
Comment (grobie) [inline test.rb]: What's the purpose of this file?

**Comment by cstyan** [comment]
removed `test.rb`, did not mean to stage that file :)

**Comment by grobie** [comment]
Thank you!

**Pull request merged**